### DISCUSSION TITLE
How to ask questions from file like excel or ppt using qwen 2.5?

### DISCUSSION THREAD

**Discussion by monksgoyal** [QwenLM/Qwen3#999] [General]
I want to upload pdf and excel and ask model to give me analysis from excel etc to user. What is the best way to do it?

**Comment by DrYurets** [comment]
How to upload a file via API?

![image](https://github.com/user-attachments/assets/59f90872-32be-4ccb-befb-7b0b55d3b7cb)

**Comment by DrYurets** [comment]
The only information I've found is:
[https://www.alibabacloud.com/help/en/model-studio/developer-reference/upload-files-by-calling-api?spm=a2c63.p38356.help-menu-2400256.d_3_9_5.7ae8a52fCuXKWF](url)
